Ambient-Sensing.md

```diff
@@ -0,0 +1,255 @@
+>
+> ```bash
+> # Delete device:
+> http DELETE http://localhost:59881/api/v2/device/name/pluto-dht22
+>
+> # Delete profile, if modified:
+> http DELETE http://localhost:59881/api/v2/deviceprofile/name/temperature-humidity-sensor
+>
+> # Restart:
+> sudo snap restart edgex-device-usb-camera
+> ```
+
+## Querying stored data from EdgeX
+Let's query EdgeX Core Data to check if measurements (readings) are being added via Device MQTT. We use the `readings` endpoint and query just 2 records:
+```bash
+http http://localhost:59880/api/v2/reading/device/name/pluto-dht22?limit=2 --body
+```
+```json
+{
+    "apiVersion": "v2",
+    "readings": [
+        {
+            "deviceName": "pluto-dht22",
+            "id": "94e2fbf8-4da5-403b-97a0-6b9cd7fab00e",
+            "origin": 1655806229236796266,
+            "profileName": "temperature-humidity-sensor",
+            "resourceName": "humidity",
+            "value": "4.120000e+01",
+            "valueType": "Float32"
+        },
+        {
+            "deviceName": "pluto-dht22",
+            "id": "b5588876-cef9-4fde-bf29-87ab0a92f187",
+            "origin": 1655806229234657153,
+            "profileName": "temperature-humidity-sensor",
+            "resourceName": "temperature",
+            "value": "2.560000e+01",
+            "valueType": "Float32"
+        }
+    ],
+    "statusCode": 200,
+    "totalCount": 16
+}
+```
+
+## Visualizing sensor data with Grafana
+We use Grafana to query the readings from EdgeX Core Data. We use a Grafana plugin called [JSON API](https://grafana.com/grafana/plugins/marcusolsson-json-datasource/)to query and pick the needed information.
+
+#### Install
+```bash
+sudo snap install grafana --channel=rock/edge
+sudo snap start --enable grafana
+```
+Open UI: http://jupiter.local:3000
+
+Default username/password: admin/admin
+
+#### Install JSON API Plugin
+Install the [JSON API](http://localhost:3000/plugins/marcusolsson-json-datasource?page=overview) plugin via "configuration->plugin":
+[http://localhost:3000/plugins/marcusolsson-json-datasource?page=overview](http://localhost:3000/plugins/marcusolsson-json-datasource?page=overview))
+
+#### Add a datasource
+Select JSON API and set the following parameters:
+* name: core-data  
+* URL: http://localhost:59880/api/v2/reading
+
+Save and test. You should see Not Found as follows, meaning that the server was set correctly but there is no resource at the given URL. To resolve this, we will later on set the HTTP path in the query editor.
+
+
+#### Create a dashboard
+To do so, go follow: + -> Create / Dashboard
+
+Set the query refresh rate to `5s`
+
+> ℹ **Tip**  
+> The range can be shorted by manually entering the from value such as: now-1m
+
+#### Setup the panel
+a. Add an empty panel. Set the title to Temperature.
+
+b. Setup query and transformation:
+-   Name `Pluto`
+-   Field `$.readings[:].value`, Type `Boolean`, Alias `Value`
```

Comment:
The value's type should be either string(float32) or number(int32)

Reply:
Removed the grafana section for now.